### ISSUE TITLE
About calculating dist_keypts 

### ISSUE THREAD

**Issue by houyongkuo** [XuyangBai/D3Feat.pytorch#20]
Hello, I see that you are using `dist_keypts = cdist(sel_P_src, sel_P_src)` instead of `dist_keypts = cdist(sel_P_src, sel_P_tgt)` when calculating dist_keypts, why is this? Thanks

**Comment by 2019EPWL** [comment]
yes, I found that problem too!  I think it should be `dist_keypts = cdist(sel_P_src, sel_P_tgt)` . @HOUYONGKUO @XuyangBai

**Comment by houyongkuo** [comment]
> yes, I found that problem too! I think it should be `dist_keypts = cdist(sel_P_src, sel_P_tgt)` . @HOUYONGKUO @XuyangBai

But after my test, code `dist_keypts = cdist(sel_P_src, sel_P_src)` can indeed get a better result, I don't know the reason.
